delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/slowqueries/BlazegraphSlowQueryLogger.scala

```diff
@@ -24,21 +24,22 @@ trait BlazegraphSlowQueryLogger {
     * @return
     *   the query
     */
-  def apply[E, A](context: BlazegraphQueryContext, query: IO[E, A]): IO[E, A]
+  def apply[A](context: BlazegraphQueryContext, query: IO[A]): IO[A]
 }
 
 object BlazegraphSlowQueryLogger {
 
   private val logger = Logger[BlazegraphSlowQueryLogger]
 
   def apply(sink: BlazegraphSlowQueryStore, longQueryThreshold: Duration)(implicit
-      clock: Clock[UIO]
+      clock: Clock[IO]
   ): BlazegraphSlowQueryLogger = new BlazegraphSlowQueryLogger {
-    def apply[E, A](context: BlazegraphQueryContext, query: IO[E, A]): IO[E, A] = {
-      query.attempt.timed
-        .flatMap { case (duration, outcome) =>
-          UIO
-            .when(duration >= longQueryThreshold)(logSlowQuery(context, outcome.isLeft, duration))
+    def apply[A](context: BlazegraphQueryContext, query: IO[A]): IO[A] = {
+      IOInstant
+        .timed(query.attempt)
```

Comment:
I think you have an implicit class method for this now?

Reply:
In your PR or somewhere else?

Reply:
Oh I thought .timed was an implicitly added method. Maybe it could be?